content/post/contributors/patricia_tchuegno.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Delia_pixel"
+date: 2022-01-04T13:46:09+01:00
+authors: ["Patricia Tchuegno"]
+---
+
+Hello, I'm Patricia Tchuegno, Sofware developper. I'm passionate about making beautiful websites. That's why I usually work on the front end side. But I also like writing algorithms and solving problems. So I can work on the back end side also. I have skills in digital marketing and want to learn more about Data Science. I used to code with AngularJs, ReactJs, HTML, CSS, Javascript, Jquery, Bootstrap, Laravel, etc.
+By now I really want to perform my skills in programming to add my value and have more experience.
```

Comment:
> By now I really want to perform my skills in programming to add my value and have more experience.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;^
This doesn't make much sense, you meant "perfect" maybe?